### PR TITLE
Use the system configured timezone when formatting logs

### LINKED ISSUE #1799: i run project,but log time is UTC.I want time GMT+8
i run project,but log time is UTC.I want time GMT+8 .Thanks!!

### PR DESCRIPTION
**Pull request by fantayeneh**
This PR addresses
- the issue of log formatter using hardcoded (utf) timezone formatter

Fixes #1799 

Sign the certificate of originSigned-off-by: Fantayeneh Asres Gizaw <fantayeneh@gmail.com>

### CODE REVIEW

**Comment by pcalcado** [comment]
Thanks for the patch, @fantayeneh! While we review it,  would you ming adding a DCO signature to this commit?

In practical terms, you should only have to do:
``shell
$ git commit --amend -s -a --no-edit 
```

And the force push the branch. This will add a signature to the commit message. For more info you can check this doc:

https://github.com/linkerd/linkerd/blob/master/CONTRIBUTING.md#certificate-of-origin

**Comment by fantayeneh** [comment]
@pcalcado Just did that, not sure why is not picking it up

**Comment by siggy** [comment]
@fantayeneh Sorry about the DCO Bot, it's not very helpful here. It may be the malformed message in your first commit.
```
Sign the certificate of originSigned-off-by: Fantayeneh Asres Gizaw <fantayeneh@gmail.com>
```

Perhaps try rebasing and squashing into a single commit.

File: admin/src/test/scala/io/buoyant/admin/LogFormatterTest.scala [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package io.buoyant.admin
+
+import java.util.TimeZone
+import java.util.logging.{Level, LogRecord}
+
+import com.twitter.util.TimeFormat
+import org.scalatest.WordSpec
+
+class LogFormatterTest extends WordSpec {
```
Comment (olix0r) [inline admin/src/test/scala/io/buoyant/admin/LogFormatterTest.scala]: Note that we use `FunSuite` throughout this repo. I have a minor preference to keep it consistent, but it's in no way a blocker ;)

**Comment by siggy** [comment]
Created #1840 to track subsequent work.

**Pull request merged**